chunky/src/java/se/llbit/math/SimplexNoise.java

```diff
@@ -0,0 +1,217 @@
+//
+// Simplex noise generator for Processing
+//
+// This is based on the C++ implementation written by Stefan Gustavson.
+// Translated and modified by Keijiro Takahashi. The following note is from
+// the original source code.
+//
+
+//
+// sdnoise1234, Simplex noise with true analytic
+// derivative in 1D to 4D.
+//
+// Copyright © 2003-2011, Stefan Gustavson
+//
+// Contact: stefan.gustavson@gmail.com
+//
+// This library is public domain software, released by the author
+// into the public domain in February 2011. You may do anything
+// you like with it. You may even remove all attributions,
+// but of course I'd appreciate it if you kept my name somewhere.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
```

Comment:
Is this public domain, gpl or dual-licensed? We should probably add the notice to the readme and about dialog, too.

Reply:
I've taken the code from here https://github.com/keijiro/sketches2016/blob/master/Simplex2/SimplexNoise.java which is itself a java reimplementation of the code from here https://github.com/stegu/perlin-noise/blob/master/src/sdnoise1234.c. You can see that the original code no longer has the GPL mention, see https://github.com/stegu/perlin-noise/issues/4
So the original code is definitely public domain, but I am unsure about the reimplementation. I assume that the author intended to release under the same condition but this is a bit unclear... Maybe I should try to ask the author?
As for adding them in the readme+dialog, no problem

Reply:
Clearly, the author of the Java reimplementation wanted to un-license it: https://github.com/keijiro/sketches2016#license